cvmfsInstall.sh

```diff
@@ -57,12 +58,23 @@ for dir in $(find $BASEDESTDIR/* -maxdepth 0 -type d | grep -G "20[0-9][0-9]-[0-
 
 dockerrun()
 {
-  if [ X$(echo $SCRAM_ARCH | cut -d"_" -f 1) == "Xslc7" ]; then
-    ARGS="cd $THISDIR; $@"
-    docker run -t -e THISDIR=${THISDIR} -e WORKDIR=${WORKDIR} -e SCRAM_ARCH=${SCRAM_ARCH} -e x=${x} -v ${WORKDIR}:${WORKDIR} -v ${THISDIR}:${THISDIR} -u $(whoami) cmssw/slc7-installer sh -c "$ARGS"
-  else
-    eval $@
-  fi
+  case "$SCRAM_ARCH" in
+    slc7_amd64_* )
+      ARGS="cd $THISDIR; $@"
+      docker run -t -e THISDIR=${THISDIR} -e WORKDIR=${WORKDIR} -e SCRAM_ARCH=${SCRAM_ARCH} -e x=${x} -v ${WORKDIR}:${WORKDIR} -v ${THISDIR}:${THISDIR} -u $(whoami) cmssw/slc7-installer sh -c "$ARGS"
+      ;;
+    slc7_aarch64_* )
+      ARGS="export THISDIR=${THISDIR}; export WORKDIR=${WORKDIR}; export SCRAM_ARCH=${SCRAM_ARCH}; export x=${x}; cd ${THISDIR}; $@"
```

Comment:
Actually it would be smarter to do `export THISDIR='${THISDIR}'`, just in case there is a whitespace in these variables, but that should be unlikely.